session6/src/com/itschool/homeworks6/HomeworkS6Ex1.java

```diff
@@ -0,0 +1,15 @@
+package com.itschool.homeworks6;
+
+public class HomeworkS6Ex1 {
+
+    public static void main(String[] args) {
+
+
+        String stringOne = "OpenAi";
+        String stringTwo = "OpenAi";
+        System.out.println(stringOne == stringTwo);
+
+
```

Comment:
too more empty lines here. It is a best practice to not have so much empty spaces in the code.